assets/tailwind.config.js

```diff
@@ -1,12 +1,40 @@
+let plugin = require("tailwindcss/plugin");
 module.exports = {
   content: [
-    "./js/**/*.js",
-    "./css/**/*.css",
-    "../lib/**/*.*ex",
     "../**/*.*exs",
-    "../../../config/*.*exs",
-
-    // We need to include the Palette dependency so the classes get picked up by JIT.
-    "../deps/palette/**/*.*ex"
-  ]
+    "./js/**/*.js",
```

Comment:
@gtenaschuk if we remove `"./css/**/*.css",` we won't be able to add our own tailwind classes right?

Reply:
To do that you need to include your css classes or files on the app.css file, if you want to make your new custom css priority  it should go after palette import, if you want to make palette priority your custom css should go before palette.

Reply:
Also, if you are talking about using tailwind core css that does not exist in palette, you just need to use them in your app components or in your templates, tailwind will add them to the final priv css when the deploy command is executed.